### PR TITLE
PCA polishing edits

### PR DESCRIPTION
**Pull request by jashapiro**
### Purpose

This PR contains edits and additions to the two PCA notebooks, for microarray and RNAseq.

### Issue addressed

This PR arises out of #289 and #290, editing and polishing examples for publication.

I focused mostly on adding some context in places I thought it was needed, with some edits for formatting (shortening comments and output).

There are also a couple of fixes that I noticed along the way, notably in the final plot of the RNAseq notebook, which is not included in the notebook, not just saved to disk.

#### Gotchas the reviewer should know about

None that I can think of directly. 

## Remaining concerns and questions

 I have focused here on the body of the notebook, leaving for later the introductory "Purpose" paragraph. As such, this is still mostly focused on "how" not "why".

### CODE REVIEW

File: 02-microarray/dimension-reduction_microarray_01_pca.Rmd [hunk 1]
```diff
@@ -196,14 +196,14 @@ metadata <- readr::read_tsv(metadata_file)
 
 # Read in data TSV file
 df <- readr::read_tsv(data_file) %>%
-  # Tuck away the gene ID  column as rownames
+  # Tuck away the gene ID column as rownames, leaving a numeric matrix
```
Comment (cansavvy) [inline 02-microarray/dimension-reduction_microarray_01_pca.Rmd:199]: It's still a data frame though so I don't want them confused. So is there a different way we can say this?

File: 02-microarray/dimension-reduction_microarray_01_pca.Rmd [hunk 2]
```diff
@@ -251,33 +255,36 @@ The `summary()` function reports the proportion of variance explained by each pr
...
 ## Prepare a final data frame with PCA results for plotting
 
 In the next chunk, we are going to extract the first two principal components from our `pca` object to prepare a data frame for plotting.
 
 ```{r}
-# Make the first two principal components into a data frame for plotting with `ggplot2`
+# Make the first two PCs into a data frame for plotting with `ggplot2`
 pca_df <- data.frame(pca$x[, 1:2]) %>%
-  # Turn samples_ids stored as rownames into column
+  # Turn samples ids stored as rownames into a column
```
Comment (cansavvy) [inline 02-microarray/dimension-reduction_microarray_01_pca.Rmd:275]:
```suggestion
  # Turn samples IDs stored as rownames into a column
```

To fit CONTRIBUTING.md style

File: 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd [hunk 3]
```diff
@@ -215,21 +213,20 @@ metadata <- readr::read_tsv(metadata_file)
 
 # Read in data TSV file
 df <- readr::read_tsv(data_file) %>%
-  # Here we are going to store the gene IDs as rownames so that we can have a numeric matrix to perform calculations on later
+  # Tuck away the gene ID column as rownames, leaving a numeric matrix
```
Comment (cansavvy) [inline 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd:216]: Same comment.

File: 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd [hunk 4]
```diff
@@ -238,49 +235,53 @@ Now we are going to use a combination of functions from the `DESeq2` and `ggplot
...
+We want to filter out the genes that have not been expressed or that have low expression counts because we want to reduce noise in our data before we normalize  and create our plot.
 We are going to do some pre-filtering to keep only genes with 10 or more reads total.
+This threshold might vary depending on the number of samples and expression patterns acrosss your data set.
 Note that rows represent gene data and the columns represent sample data in our dataset.
 
 ```{r}
-# Define a minimum counts cutoff and filter `DESeqDataSet` object to include
-# only rows that have counts above the cutoff
+# Filter `DESeqDataSet` object to include only genes
+# with a total count of at least 10 reads across the samples
```
Comment (cansavvy) [inline 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd:284]: Do you wanna squeeze in the #417 change while we are at it?

File: 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd [hunk 5]
```diff
@@ -291,28 +292,36 @@ We are going to use the `vst()` function from the `DESeq2` package to normalize
...
 Now we'll plot the PCA using both `refinebio_treatment` and `refinebio_disease` variables for labels since they are central to the androgen deprivation therapy (ADT) based hypothesis in the [original paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6210624/) [@Sharma2018].
 
 ```{r}
-plotPCA(dds_norm,
-  intgroup = c("refinebio_treatment", "refinebio_disease") # Note that we are able to add another variable to the intgroup argument here by providing a vector of the variable names with  `c()` function
+plotPCA(
+  dds_norm,
+  intgroup = c("refinebio_treatment", "refinebio_disease")
+  # We are able to add another variable to the intgroup argument
+  # by providing a vector of the variable names with  `c()` function
```
Comment (cansavvy) [inline 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd:324]:
```suggestion
  # by providing a vector of the variable names with `c()` function
```

**Review by cansavvy — COMMENTED** [review]
These seem like great edits! I have tiny comments besides wondering if we want to squeeze in #417 's change while we are at it.

File: 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd (hunk 4 above)
Comment (jashapiro) [reply 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd:284]: I think I'd rather leave those for a separate PR.

Comment (cansavvy) [reply 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd:284]: Okey dokey.

**Review by cansavvy — APPROVED** [review]
Looks great! 🚀

**Pull request merged**